src/main/java/org/mitre/stix/DocumentUtilities.java

```diff
@@ -35,21 +35,31 @@
 
 /**
  * A collection of utility helper methods.
- * 
+ *
  * @author nemonik (Michael Joseph Walsh <github.com@nemonik.com>)
  */
 public class DocumentUtilities {
 
 	private static final String XML_SCHEMA_INSTANCE = "http://www.w3.org/2001/XMLSchema-instance";
 	private static final String XML_NAMESPACE = "http://www.w3.org/2000/xmlns/";
 
+	private static final JAXBContext STIX_CONTEXT = initDefaultContext();
+
+	private static JAXBContext initDefaultContext(){
+		try {
+			return JAXBContext.newInstance("org.mitre.stix.stix_1");
```

Comment:
It should work. `STIXPackage` is the most general container that comprises all other types in `org.mitre`, and it is in `org.mitre.stix.stix_1`. So when JAXBContext is initialised it should scan all the types in `org.mitre.stix.stix_1` and all other types that are referenced by STIXPackage recursively and irrespectively to the initial package. So all types in full `org.mitre` are in the context. Please let me know if I'm wrong, because I'm not an expert in JAXB. The unit tests which construct STIXPackage with cybox contents and then invoke `toDocument` in the project would probably fail if I'm wrong.